### ISSUE TITLE
Sleet unable to find 2.1.0-rc1 installation

### ISSUE THREAD

**Issue by phil-harmoniq** [emgarten/Sleet#37]
`sleet --help`

Output:
```
It was not possible to find any compatible framework version
The specified framework 'Microsoft.NETCore.App', version '2.1.0-rc1-26428-03' was not found.
  - Check application dependencies and target a framework version installed at:
      /usr/local/share/dotnet
  - Installing .NET Core prerequisites might help resolve this problem:
      http://go.microsoft.com/fwlink/?LinkID=798306&clcid=0x409
  - The .NET Core framework and SDK can be installed from:
      https://aka.ms/dotnet-download
  - The following versions are installed:
      2.0.5 at [/usr/local/share/dotnet/shared/Microsoft.NETCore.App]
      2.1.0-preview1-26216-03 at [/usr/local/share/dotnet/shared/Microsoft.NETCore.App]
      2.1.0-preview2-26406-04 at [/usr/local/share/dotnet/shared/Microsoft.NETCore.App]
      2.1.0-rc1 at [/usr/local/share/dotnet/shared/Microsoft.NETCore.App]
```

As you can see, I *do* have `2.1.0-rc1` installed but it seems your application requires a very specific rc1 build. The currently released version of the the 2.1 SDK is `2.1.300-rc1-008673`

**Comment by emgarten** [comment]
What steps did you use to install the rc build?

Does changing the version in  `$(HOME)/.dotnet/tools/.store/sleet/2.3.21/sleet/2.3.21/tools/netcoreapp2.1/any/Sleet.runtimeconfig.json` help?

**Comment by phil-harmoniq** [comment]
I installed it using `dotnet tool install -g sleet`. Changing the version tag to `2.1.0-rc1` in `Sleet.runtime.json` does seem to fix the issue.

**Comment by emgarten** [comment]
Good to know that fixes it. With so many different versions of 2.1.0 betas right now I'm not sure that there will be a good fix for this until a stable release comes out.

**Comment by emgarten** [comment]
This should be fixed in the latest build which targets 2.1.300 RTM
